docs/getting-started-guides/aws/public.md

```diff
@@ -59,6 +59,7 @@ for i in $(ls .terraform/modules/*/Makefile); do i=$(dirname $i); make -C $i; do
 ### Provision the cluster infrastructure
 
 ```
```

Comment:
I spoke too soon.  It needs to stay.

Explanation is from 
https://github.com/Capgemini/kubeform/commit/337e104d0704f5e0a5bae72628f4645572543ca3

> In Terraform 0.7.x, template file data sources do not support the
> 'provisioner' parameter. So instead, we must now use a null_resource to
> get the discovery URL.
> 
> And, unfortunately, depending on a null_resource does not mean that you
> can read the file before it exists.
>